proto/buoyant-cloud-api.proto

```diff
@@ -133,4 +155,5 @@ service Api {
   rpc WorkloadStream(stream WorkloadMessage) returns (Empty) {}
   rpc AddEvent(Event) returns (Empty) {}
   rpc LinkerdInfo(LinkerdMessage) returns (Empty) {}
+  rpc ManageAgent(Auth) returns (stream AgentCommand) {}
```

Comment:
Yup `Auth` is sufficient.

In hindsight, I'm wondering if we should have just used basic auth, rather than putting an `Auth` into each API call. Something to consider longer term.